packages/driver/test/cypress/integration/commands/waiting_spec.js

```diff
@@ -1,13 +1,23 @@
 const { _, Promise } = Cypress
+let reqQueue = []
 
 const xhrGet = function (win, url) {
-  const xhr = new win.XMLHttpRequest
+  const xhr = new win.XMLHttpRequest()
 
   xhr.open('GET', url)
 
+  reqQueue.push(xhr)
+
   return xhr.send()
 }
 
+// keep track of outstanding requests and abort them so
```

Comment:
fun fact, the new network stubbing will terminate unclosed intercepted requests before each test, so this may become unnecessary one day

Reply:
Great!